specs/WindowControlsOverlayConfiguration.md

```diff
@@ -0,0 +1,142 @@
+WebView2 Window Controls
+===
+
+# Background
+This API allows you to Enable and configure The Webview2 Window Controls overlay.
+The Overlay is a region on the top right/left of the webview window which contains 
+the caption buttons (minimize, maximize, restore, close). Enabing the Overlay allows
+for custom app title bars rendered completly inside the webview window.
+The overlay Settings lives on the controller object.
+
+This API is designed to be used in addition with the other non-client region APIs 
+and features. These include `app-region: drag`, and `IsNonClientRegionSupportEnabled`.
+# Examples
+
+## Win32 C++
+```cpp 
+AppWindow::AppWindow() {
+    m_mainWindow = CreateWindowExW(
+        WS_EX_CONTROLPARENT, 
+        GetWindowClass(),
+        L""
+        WS_POPUPWINDOW, 
+        0,0, 800, 800, 
+        nullptr, nullptr, 
+        g_hInstance, nullptr);
+}
+
+void  AppWindow::OnCreateWebview2ControllerCompleted(HRESULT hr, ICoreWebview2Controller* controller) 
+{
+    wil::com_ptr<ICoreWebView2Controller5> controller5;
+    CHECK_FAILURE(controller->QueryInterface(&controller5));
+
+    wil::com_ptr<ICoreWebView2WindowControlsOverlaySettings> wco_config;
+    CHECK_FAILURE(controller5->get_WindowControlsOverlaySettings(&wco_config));
+
+    wco_config->put_IsEnabled(true);
```

Comment:
Do you need to check for failure of HRESULT return value from put_IsEnabled and put_TitleBarColor calls?

Reply:
added